apps/web/components/booking/pages/BookingPage.tsx

```diff
@@ -176,6 +176,7 @@ const BookingPage = ({
           email: attendees[0].email,
           location,
           eventName: profile.eventName || "",
+          bookingId: id,
```

Comment:
@joeauyeung This was missing from the query params when it came to a recurring event, fixed